package.json

```diff
@@ -16,6 +16,8 @@
     "lint": "eslint --ext .js,.ts . --cache --report-unused-disable-directives",
     "test:ci:cocoapods": "ruby packages/platform-ios/native_modules.rb",
     "postinstall": "yarn build",
+    "link-packages": "PACKAGES=('tools' 'platform-ios' 'platform-android' 'cli' 'cli-types' 'global-cli') && cd packages && for PACKAGE in \"${PACKAGES[@]}\"; do cd $PACKAGE && yarn link && cd ..; done",
+    "unlink-packages": "PACKAGES=('tools' 'platform-ios' 'platform-android' 'cli' 'cli-types' 'global-cli') && cd packages && for PACKAGE in \"${PACKAGES[@]}\"; do cd $PACKAGE && yarn unlink && cd ..; done",
```

Comment:
I don't see this necessary. `yarn link` only marks a dependency as "linkable", no need to unlink these. Also, please remove `global-cli` here and in `link-packages`, we don't want anybody to touch it for now.

Reply:
Could be useful in a very few cases where you have several clone of the cli on your machine? 😅 Ok, very small chance.
Do you still want me to remove it? 🙂